src/iidm/Network.cpp

```diff
@@ -174,6 +174,15 @@ const stdcxx::DateTime& Network::getCaseDate() const {
     return m_caseDate;
 }
 
+stdcxx::const_range<Connectable> Network::getConnectables() const {
+    return m_networkIndex.getAll<Connectable>();
+}
+
+stdcxx::range<Connectable> Network::getConnectables() {
+    const auto& mapper = map<Connectable>;
+    return static_cast<const Network*>(this)->getConnectables() | boost::adaptors::transformed(mapper);
```

Comment:
Is it possible to write `return m_networkIndex.getAll<Connectable>();` ?

Reply:
Yes it is